### PR TITLE
Simplify `F.sigmoid` test

### PR DESCRIPTION
**Pull request by crcrpar**
This PR aims at rewriting sigmoid function test.

rel: #6071

### CODE REVIEW

**Comment by toslunar** [comment]
Jenkins, test this please.

File: tests/chainer_tests/functions_tests/activation_tests/test_sigmoid.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 import chainer
 from chainer.backends import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
+from chainer import utils
```
Comment (toslunar) [inline tests/chainer_tests/functions_tests/activation_tests/test_sigmoid.py]: Could you sort imports?

Comment (crcrpar) [reply tests/chainer_tests/functions_tests/activation_tests/test_sigmoid.py]: Is this OK?
```python
from chainer import testing
from chainer.testing import attr
from chainer import utils
```

Comment (toslunar) [reply tests/chainer_tests/functions_tests/activation_tests/test_sigmoid.py]: Yes.  OpenStack Guideline recommends ordering imports by the full module paths.  https://docs.openstack.org/hacking/latest/user/hacking.html#imports

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failures are unrelated to the PR.

**Pull request merged**